test/dummy/config/letsencrypt_plugin.yml

```diff
@@ -1,7 +1,7 @@
 default: &default
   endpoint: 'https://acme-staging.api.letsencrypt.org/directory'  # test server
   email: 'your@email.address'
-  domain: '<%= ['example.com', 'www.example.com', 'another.example.com'].join(" ") %>'
+  domain: "<%= ['example.com', 'www.example.com', 'another.example.com'].join(" ") %>"
```

Comment:
Thanks! Would you also change double quotes to single ones in .join() at the end of this line (I can't check it right now but I think it would be needed to)?